src/Core/src/Platform/iOS/SwitchExtensions.cs

```diff
@@ -7,6 +7,9 @@ namespace Microsoft.Maui.Platform
 {
 	public static class SwitchExtensions
 	{
+		static UIColor? OffTrackColor;
+		static bool HasSwitched;
```

Comment:
How does this static field work with multiple controls?

Reply:
It works okay with multiple controls since share this default off track color. But there are some discrepancies with Dark mode which I am addressing now!